backend/app/assets/javascripts/spree/backend/stock_transfer.js.coffee

```diff
@@ -1,196 +0,0 @@
-$ ->
```

Comment:
Having some trouble finding where this code was moved to. Was this file deleted by mistake?